lib/corefoundation.rb

```diff
@@ -1,13 +1,26 @@
-require 'ffi'
-require 'iconv' if RUBY_VERSION < "1.9"
+require "ffi" unless defined?(FFI)
 
-require 'corefoundation/base'
-require 'corefoundation/null'
-require 'corefoundation/string'
-require 'corefoundation/array'
-require 'corefoundation/boolean'
-require 'corefoundation/data'
-require 'corefoundation/dictionary'
-require 'corefoundation/number'
-require 'corefoundation/date'
-require 'corefoundation/extensions'
+require "corefoundation/base_wrapper"
```

Comment:
These can all be converted to require_relative which is a bit faster in ruby. It avoids walking the gem path a bunch.